source/release-notes/1.6.txt

```diff
@@ -42,6 +42,50 @@ For more information, see :ref:`c2c-filter-regex`.
 
 Starting in 1.6.0, ``mongosync`` supports sync with MongoDB 7.0.
 
+Other Notes
+~~~~~~~~~~~
+
+New Features:
+
+- Sped up the commit process by parallelizing metadata cleanup. 
+  (:issue:`REP-2568`)
+
+- Reduced memory allocation during change event application by ~30% for 
+  pre-6.0 releases of MongoDB Server versions. (:issue:`REP-3015`)
+
+- Changed telemetry to also send error messages to Segment when ``mongosync``
+  exits with a fatal error. (:issue:`REP-3138`) 
+
+- Added support for ``mongosync`` to sync between source and destination 
+  clusters that have the same major and minor versions, but different 
+  patch versions. (:issue:`REP-3161`)
+
+- Upgraded build to use Go 1.20. (:issue:`REP-3044`)
+
+- ``mongosync`` now periodically logs when server operations take a longer 
+  than expected time. (:issue:`REP-3176`)
+
+Issues Fixed:
+
+- Fixed bug introduced in 1.5, where ``mongosync`` could fail to apply change
+  events for a document if they occur shortly after the document is copied during
+  the Collection Copy phases. (:issue:`REP-3253`)
+
+- Fixed bug where ``mongosync`` could crash due to source cluster containing
+  legacy indexes with unknown index options. (:issue:`REP-2885`)
+
+- Fixed a bug where mongosync could crash with an error during change event 
+  application due to its failing to read 500 documents within 5 minutes or 
+  reaching the end of the oplog when reading from change streams on the source
+  cluster. (:issue:`REP-3012`)
+
+- Fixed bug where ``mongosync`` could crash with error when indexes apply
```

Comment:
I believe we want to document more information on how this bug can be triggered to let users figure out how serious the bug is. @autarch Maybe we could shortern the reproduction steps from the release [notes](https://docs.google.com/document/d/1vDjeXw8x592d3NLKL5RHL6Nouw64heAL-yod_PyqKME/edit#bookmark=id.9eacpwp2gh2c)?

Reply:
We could say something like this:

> This crash occurred when such an index was added or dropped at nearly the same time as an index that violated these constraints was deleted or inserted. It would manifest with an error like "Ambiguous field name found in array (do not use numeric field names in embedded elements in an array)" or a similar error for other index types.
>
> This fix makes this crash less likely, but it does not eliminate the possibility that it will occur.